encoding/descaling.md

```diff
@@ -1,477 +1,365 @@
+and no obvious effects or filtering applied to it
+(like for example blur, glow, or heavy grain).
+Here is another example of such a frame:
+
+![](images/descale/descaled_illya_02_frame.png)
+
+![Result from the default settings](images/descale/descaled_illya_02_graph.png)
+
+If you've got a graph without any clear spikes,
+it's usually a bad idea to try descaling that.
+It's likely that you're either working with native 1080p content,
+an already-descaled encode,
+or a source that has been warped too much to properly descale.
+
+![](images/descale/descaled_fgo_op_encoded_frame.png)
+
+![Running getnative on frame of a already-rescaled encode](images/descale/descaled_fgo_op_encoded_graph.png)
+
+Also of note is that you should be careful
+to not pick a frame with credits to run getnative on.
+Credits are usually added at the very end,
+after the clip has already been upscaled.
+Those will need to be handled separately.
+
+![](images/descale/descaled_yaiba_17_credits_frame.png)
+
+![Credits and other native 1080p elements mess with the graph](images/descale/descaled_yaiba_17_credits_graph.png)
+
+There is also letterboxing.
+When talking about letterboxing,
+we refer to the black bars
+you occasionally find around a video.
+Running these through getnative will result in very odd graphs
+that you can't discern much from.
 
-[getnative]: https://github.com/Infiziert90/getnative
-[descale]: https://github.com/BluBb-mADe/vapoursynth-descale
+![](images/descale/descaled_crystar_op1_letterbox_frame.png)
+
+![There's no way to figure out the native resolution from this](images/descale/descaled_crystar_op1_letterbox_graph.png)
 
+And finally, we run into heavily post-processed videos.
+These are usually impossible to accurately descale
+due to additional post-processing that was applied during the scaling,
+or due to the scaler used being so destructive
+that it leaves no detail to preserve.
 
-## Finding out the native resolution
+![](images/descale/descale_paniponidash_op2_qtec_frame.png)
 
-To figure out what
-the native resolution of an anime is,
-first you need a good frame to test.
-Ideally,
-you'll want a bright frame
-with as little blur as possible of high quality
-(Blu-ray or very good webstreams).
-It also helps to not have
-too many post-processed elements in the picture.
-Whilst it is most definitely possible
-to get pretty good results with "bad" frames,
-it's generally better to use
-good frames whenever possible.
+![This is what a murder scene looks like](images/descale/descale_paniponidash_op2_qtec_graph.png)
 
-Here are some examples of "bad" frames.
+[getnative]: https://github.com/Infiziert90/getnative
 
-![Manaria Friends — 01 (frame 1)](images/descale_manaria01.png)
 
-This picture is dark.
-It also has some effects over it.
+# Filterchains
 
-![Manaria Friends — 01 (frame 2)](images/descale_manaria02.png)
+Once you've figured out the native resolution,
+you can move on to the actual descaling.
+There's two common ways to do this.
```

Comment:
```suggestion
There's three common ways to do this.
```
Whoops, didn't notice that when I added mention of the python functions.

Reply:
Should those even be separate entries?